Missing relationship surveys_campaigns

<!--- Provide a general summary of the issue in the **Title** above -->
<!--- Before you open an issue, please check if a similar issue already exists or has been closed before. --->
<!--- If you have discovered a security risk please report it by emailing security@suitecrm.com. This will be delivered to the product team who handle security issues. Please don't disclose security bugs publicly until they have been handled by the security team. --->

#### Issue
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->


There is a missing relationship in SuiteCRM.

```
[ERROR] Unable to find relationship surveys_campaigns
```

The relationship surveys_campaigns is not defined:

```
/m/d/t/SuiteCRM git:develop ❯❯❯ git grep surveys_campaigns                                                                                                                                                                    
modules/Campaigns/vardefs.php:                'relationship' => 'surveys_campaigns',
```


#### Expected Behavior
<!--- Tell us what should happen -->

The relationship should be defined

#### Actual Behavior
<!--- Tell us what happens instead -->
<!--- Also please check relevant logs (suitecrm.log, php error.log etc.) -->

There are several ERRORS en suitecrm log

#### Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->

#### Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug include code to reproduce, if relevant -->
1. Create a Campaign
2. Open Campaign Wizard
3. Verify ERRORS en suitecrm log


#### Context
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- If you feel this should be a low/medium/high priority then please state so -->

#### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* SuiteCRM Version used: 7.10.10
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 63
* Environment name and version (e.g. MySQL, PHP 7): PHP 5.6
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 18.10

--- a/modules/Campaigns/vardefs.php
+++ b/modules/Campaigns/vardefs.php
@@ -330,6 +330,11 @@ $dictionary['Campaign'] = array('audited' => true,
             'type' => 'index',
             'fields' => array('name')
         ),
+        array(
+            'name' => 'idx_survey_id',
+            'type' => 'index',
+            'fields' => array('survey_id')
+        ),
     ),
 
     'relationships' => array(

--- a/modules/Surveys/vardefs.php
+++ b/modules/Surveys/vardefs.php
@@ -104,7 +104,7 @@ $dictionary['Surveys'] = array(
                 'include' => 'modules/Surveys/Utils/utils.php'
             ),
         ),
-        
+
         'submit_text'              => array(
             'required'   => false,
             'name'       => 'submit_text',
@@ -171,6 +171,15 @@ $dictionary['Surveys'] = array(
         'surveys_surveyresponses' => array(
             'rhs_module'        => 'SurveyResponses',
             'rhs_table'         => 'surveyresponses',
+            'rhs_key'           => 'survey_id',
+            'lhs_module'        => 'Surveys',
+            'lhs_table'         => 'surveys',
+            'lhs_key'           => 'id',
+            'relationship_type' => 'one-to-many',
+        ),
+        'surveys_campaigns' => array(
+            'rhs_module'        => 'Campaigns',
+            'rhs_table'         => 'campaigns',
             'rhs_key'           => 'survey_id',
             'lhs_module'        => 'Surveys',
             'lhs_table'         => 'surveys',

